controllers/virtualmachine/v1alpha1/virtualmachine_controller.go

```diff
@@ -472,6 +472,16 @@ func (r *Reconciler) ReconcileNormal(ctx *context.VirtualMachineContext) (reterr
 	// Add this VM to prober manager if ReconcileNormal succeeds.
 	r.Prober.AddToProberManager(ctx.VM)
 
+	// Back up this VM if ReconcileNormal succeeds and the FSS is enabled.
+	if lib.IsVMServiceBackupRestoreFSSEnabled() {
+		err := r.VMProvider.BackupVirtualMachine(ctx, ctx.VM)
+		if err != nil {
+			ctx.Logger.Error(err, "Failed to backup VirtualMachine")
+			r.Recorder.EmitEvent(ctx.VM, "Backup", err, false)
+			return err
```

Comment:
Should we consider emitting a condition for this?  Can you file a story so we discuss that?  Thanks!